### PR TITLE
Issues/146

### LINKED ISSUE #146: Wallet improvement
https://github.com/kaguraswap/monorepo/discussions/142#discussioncomment-3105512

https://github.com/kaguraswap/monorepo/discussions/142#discussioncomment-3105567

https://github.com/kaguraswap/monorepo/discussions/142#discussioncomment-3105583

https://github.com/kaguraswap/monorepo/discussions/142#discussioncomment-3105663

### PR DESCRIPTION
**Pull request by ksuhara**
close #146

### CODE REVIEW

**Comment by taijusanagi** [comment]
I don't think this should be like this.

Screen Shot 2022-07-22 at 23 38 22

These are social media links so I think it should not be in the drop-down menu.
I thought there is some function-related things in the menu.

**Comment by taijusanagi** [comment]
I prefer to have a top page and navigate user access social icon from the top page.

And for the assets and asset page, just keep it as simple as possible.

File: packages/next-app/src/components/organisms/Connect/Connect.tsx [hunk 1]
```diff
@@ -18,12 +18,22 @@ export const Connect: React.FC<ConnectProps> = ({ buttonProps }) => {
         Connect Wallet
       </Button>
       <Modal onClose={onClose} isOpen={isOpen}>
-        <Stack spacing="4">
-          <Button width="100%" rounded="xl" onClick={() => connect({ connector: injectedConnector })}>
+        <Stack spacing="4" px="8">
+          <Button
+            width="100%"
+            rounded="xl"
+            leftIcon={<Image src={`/icons/metamask.svg`} alt="metamask-logo" width="6" />}
```
Comment (taijusanagi) [inline packages/next-app/src/components/organisms/Connect/Connect.tsx:25]: can you move to 

```
/icons/wallets/metamask.svg
```

like other icons?

File: packages/next-app/src/components/organisms/Connect/Connect.tsx [hunk 2]
```diff
@@ -18,12 +18,22 @@ export const Connect: React.FC<ConnectProps> = ({ buttonProps }) => {
...
+            onClick={() => connect({ connector: injectedConnector })}
+          >
             Metamask
           </Button>
           {/* TODO: Wallet Connect */}
```
Comment (taijusanagi) [inline packages/next-app/src/components/organisms/Connect/Connect.tsx:30]: please remove the TODO comment after implementation.

File: packages/next-app/src/lib/wagmi/index.ts [hunk 3]
```diff
@@ -12,4 +10,11 @@ export const wagmiClient = createClient({
 
 export const injectedConnector = new InjectedConnector();
 
-// export const walletConnectConnector = new WalletConnectConnector({});
+const chains = defaultChains;
```
Comment (taijusanagi) [inline packages/next-app/src/lib/wagmi/index.ts:13]: Is this config ok?

We are using own network configs, and I'm not sure wallet connects support all of chain we supports.

I prefer to get chains from configs/networks.json and set this wallet connect config

```
Object.entries(networks).map(([chainId, {rpc}])=>{
// create wallet connect config using info from network config
})
```

**Comment by ksuhara** [comment]
> I prefer to have a top page and navigate user access social icon from the top page.
> 
> And for the assets and asset page, just keep it as simple as possible.

OK currently we don't have any feature so I will remove menu button itself for now

**Comment by ksuhara** [comment]
understood and modified!

**Pull request merged**